cJSON.c

```diff
@@ -2410,3 +2410,70 @@ void cJSON_Minify(char *json)
     /* and null-terminate. */
     *into = '\0';
 }
+
+const char *cJSON_GetObjectCstr(const cJSON *object, const char *string)
+{
+	cJSON *cj = cJSON_GetObjectItem(object, string);
+
+	if (!cj || cj->type != cJSON_String || !cj->valuestring) return NULL;
```

Comment:
thanks for the pointer... no problem on not taking these.. we can just keep these in our code anyways.